src/main/java/com/employee/dao/EmployeeDAOImpl.java

```diff
@@ -12,18 +12,26 @@
 
 public class EmployeeDAOImpl implements EmployeeDAO {
 
+	public static Connection conn;
```

Comment:
Line no.15 to 21, they should be private.

Reply:
I have implemented the necessary changes based on your valuable suggestions. Would you kindly consider reviewing the updated work? Your feedback is greatly appreciated. Thank you!

Reply:
@YeWin 
 I have implemented the necessary changes based on your valuable suggestions. Would you kindly consider reviewing the updated work? Your feedback is greatly appreciated. Thank you!